message_types/status.proto

```diff
@@ -1,128 +1,162 @@
 syntax = "proto3";
 
 message Status {
+    message Site {
+        message InverterStatus {
+            string enabled = 1;
+            string connected = 2;
+        }
+
+        float productionWatt = 1;
+        float sizeLimitWatt = 2;
+        InverterStatus inverterStatus = 3;
+    }
+    
+    message OptimizersStatus {
+        int32 total = 1;
+        int32 online = 2;
+    }
+
+    enum ManagerStatus {
+        SHUTTING_DOWN = 0;
+        ERROR = 1;
+        STANDBY = 2;
+        PAIRING = 3;
+        POWER_PRODUCTION = 4;
+        AC_CHARGING = 5;
+        NOT_PAIRED = 6;
+        NIGHT_MODE = 7;
+        GRID_MONITORING = 8;
+        IDLE = 9;
+    }
 
-  message Site {
-    message InverterStatus {
-      string enabled = 1;
-      string connected = 2;
+    message Inverters {
+        message Unit {
+            string dspSn = 1;
+            float voltage = 2;
+            OptimizersStatus optimizersStatus = 3;
+            Temperature temperature = 4;
+            int32 fan = 5;
+            string errorCode = 6;
+            int32 subSystem = 7;
+            string badPosition = 8;
```

Comment:
I also get this in my response:

```
    9 {
      1 {
        1: 5000
        2: 100
      }
      2 {
        1: 1100000
        2: 100
      }
      3: 10
    }
    10 {
      1: 236143
      2: 100
    }
```

9.2.1 (`1100000`) there equals the value shown in `#/commissioning/maintenance/diagnostics/isolation-status` - a value of `11000 kOhm`

I don't know what the other values are.

Reply:
9.2.2 might be the "no fault found"

Reply:
9 is for isolation.
9.1 is for "Alpha" which I dont know what it stands for.
9.2.2 is scaling of isolation.
9.3 is fault location.

I dont know what 10.1 could be.

I have added isolation to the status proto.

Thank you for reporting this and the other points.